idea-plugin/src/test/kotlin/arrow/meta/ide/plugins/proofs/annotators/GivenAnnotatorTest.kt

```diff
@@ -70,4 +102,14 @@ class GivenAnnotatorTest : IdeTestSetUp(
 
   private val ignoredHighlightingAnnotator: List<Int>
     get() = listOf(Pass.LINE_MARKERS, Pass.EXTERNAL_TOOLS, Pass.POPUP_HINTS, Pass.UPDATE_FOLDING, Pass.WOLF)
-}
\ No newline at end of file
+}
+
+fun HighlightInfo.assertAnnotation(
```

Comment:
please add the return type, rename the function to `matches` and make the function either internal or private